propertyestimator/client.py

```diff
@@ -87,10 +127,38 @@ class PropertyEstimatorSubmission(TypedBaseModel):
     force_field: openforcefield.typing.engines.smirnoff.ForceField
         The force field parameters used during the calculations.
     """
-    properties: List[Any] = []
-    options: PropertyEstimatorOptions = None
+    def __init__(self, properties=None, force_field=None, options=None):
+        """Constructs a new PropertyEstimatorSubmission object.
+
+        Parameters
+        ----------
+        properties: list of PhysicalProperty
+            The list of physical properties to estimate.
+        options: PropertyEstimatorOptions
+            The options which control how the `properties` are estimated.
+        force_field: openforcefield.typing.engines.smirnoff.ForceField
+            The force field parameters used during the calculations.
+        """
+        self.properties = properties or []
```

Comment:
Interesting design. Never seen that before.

Reply:
It's pretty handy - I'm using it here to ensure that there isn't any mutable default parameters!